Implement modifiable user profile

NikolasGeorgaklis opened the pull request:
User is now able to change their profile picture, display name, grade, and major using the fields in editprofileviewcontroller. User info saves to parse and is updated in profileviewcontroller. Profile picture changing feature has occasional bug where it says the file name is too long.

NikolasGeorgaklis committed:
Implement modifiable user profile

User is now able to change their profile picture, display name, grade, and major using the fields in editprofileviewcontroller. User info saves to parse and is updated in profileviewcontroller. Profile picture changing feature has occasional bug where it says the file name is too long.

NikolasGeorgaklis committed:
Fix profile picture selection bug and modified editprofileviewcontroller fields to always contain most recent user info

Previously, I had a bug where I could not change the profile picture because I was using the file name provided by the phone which was too long. I changed it so that each selected image file is just named "image.png" when stored in parse. I also had it so that whenever my editprofileviewcontroller showed, the fields were automatically set to empty which was problematic. I fixed that by setting the fields to retrieved info from parse in the viewdidload in the editprofileviewcontroller.

NikolasGeorgaklis committed:
Change tab bar item icons

NikolasGeorgaklis committed:
Removed commented code in EditProfileViewController.h

NikolasGeorgaklis committed:
Create TableViews for InfoViewController and HomeViewController

InfoViewController and HomeViewController will both display a feed and therefore require tableviews.

NikolasGeorgaklis committed:
Add text elements to tableview cells in HomeViewContoller and InfoViewController

Each of these tableviews will display information in the form of text. So it is necessary that I add label elements to each of the tableview cells.

vaguilar commented on NikoBracket/InfoViewController.m:
```diff
@@ -0,0 +1,48 @@
+}
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
```
remove commented code

vaguilar commented:
everything else looks good!

NikolasGeorgaklis committed:
Add cocoa pods necessary for using an API and add refresh animation to tableviews

I added the AFNetworking and BDBOAuth1Manager pods to my podfile. I also added a refresh animation to my tableviews which doesn't do anything yet because my tableviews are not yet pulling data from an API.

NikolasGeorgaklis committed:
Remove commented code.
